### PR TITLE
[PREVIEW] DIV-2977 & DIV-3280 - Apply for DN logic + save&close page

### PR DESCRIPTION
**Pull request by johnoneill46**
# Description

Created new page for users to be brought to when the click "Save & Close"
Also updated "Apply for DN" question logic - the answer (Yes/No) is used to direct what the next step in the workflow is.

https://tools.hmcts.net/jira/browse/DIV-2977
https://tools.hmcts.net/jira/browse/DIV-3280

### CODE REVIEW

File: steps/application-saved-exit/ApplicationSavedExit.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "en": {
+    "title": "Your application has been saved",
+    "aLinkHasBeenSent": "A link to your application has been sent to:",
+    "petitionerEmail": "{{ session.petitionerEmail }}",
```
Comment (dtaylor7) [inline steps/application-saved-exit/ApplicationSavedExit.json]: just put this directly in template

File: steps/apply-for-decree-nisi/ApplyForDecreeNisi.step.js [hunk 2]
```diff
@@ -37,9 +37,14 @@ class ApplyForDecreeNisi extends Question {
   }
 
   next() {
+    if (this.fields.applyForDecreeNisi.value === 'no') {
```
Comment (dtaylor7) [inline steps/apply-for-decree-nisi/ApplyForDecreeNisi.step.js]: take a look at branches https://github.com/hmcts/one-per-page/blob/master/examples/test-app/steps/Country.step.js#L40

Comment (dtaylor7) [reply steps/apply-for-decree-nisi/ApplyForDecreeNisi.step.js]: should it redirect if answer is no? or would there be a new link under continue called `save and close`?

Comment (johnoneill46) [reply steps/apply-for-decree-nisi/ApplyForDecreeNisi.step.js]: In this instance it should be redirected when the user clicks 'no' - as per DIV-2977

**Pull request closed without merging**